config/checkstyle/checkstyle.xml

```diff
@@ -27,6 +27,12 @@
                       value="Line has leading tab characters; indentation should be performed with spaces only." />
             <property name="ignoreComments" value="true" />
         </module>
+        <module name="com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck">
```

Comment:
Could we use the `IllegalImportCheck`'s `illegalClasses` property instead? I think the regex here will also work, but it just seems more straightforward to use the IllegalImportCheck. I'm not sure of its behavior when importing a static method from one of the `illegalClasses`.
https://checkstyle.org/config_imports.html#IllegalImport

Reply:
@shakuzen Thanks for the feedback 👍 I tried it with a wrong value before and just assumed that it doesn't work with static imports. I confirmed that it works and updated in 903716d as you suggested.